tests/fixtures/key.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "did:key:zUC7GMwWWkA5UMTx7Gg6sabmpchWgq8p1xGhUXwBiDytY8BgD6eq5AmxNgjwDbAz8Rq6VFBLdNjvXR4ydEdwDEN9L4vGFfLkxs8UsU3wQj9HQGjQb7LHWdRNJv3J1kGoA3BvnBv#zUC7GMwWWkA5UMTx7Gg6sabmpchWgq8p1xGhUXwBiDytY8BgD6eq5AmxNgjwDbAz8Rq6VFBLdNjvXR4ydEdwDEN9L4vGFfLkxs8UsU3wQj9HQGjQb7LHWdRNJv3J1kGoA3BvnBv",
+  "@context": "https://w3id.org/security/multikey/v1",
+  "type": "Multikey",
+  "controller": "did:key:zUC7GMwWWkA5UMTx7Gg6sabmpchWgq8p1xGhUXwBiDytY8BgD6eq5AmxNgjwDbAz8Rq6VFBLdNjvXR4ydEdwDEN9L4vGFfLkxs8UsU3wQj9HQGjQb7LHWdRNJv3J1kGoA3BvnBv",
+  "publicKeyMultibase": "zUC7GMwWWkA5UMTx7Gg6sabmpchWgq8p1xGhUXwBiDytY8BgD6eq5AmxNgjwDbAz8Rq6VFBLdNjvXR4ydEdwDEN9L4vGFfLkxs8UsU3wQj9HQGjQb7LHWdRNJv3J1kGoA3BvnBv",
+  "secretKeyMultibase": "z488vexJQSQ2rF5GrCT8qhzGR7ASSj5rx6CtZjKNFq183woF"
```

Comment:
@aljones15 we may want to document why having this here is OK (assuming it is...) because that key name will certainly freak people (and bots...) out at some point.

Reply:
Right. I've seen some of those before. Mostly just thinking we should document why this is setup this way--vs. using hidden secret keys--'cause it looks odd/unexpected.

Reply:
The secretKey (at least in this case) is not hidden, it reuses existing values use in other suites, and the format of the key is the format of exported Multikeys.

Reply:
All good. Just thinking someone at some point is gonna express concern about "secrets" stored in the commit history...and we should be prepared to explain why that's OK in this case (across all the test suites). Nothing needs to change here, just something we need to be ready to field.

Reply:
so the local issuer PR uses the key here: https://github.com/w3c/vc-di-bbs-test-suite/pull/16